code/edge/src/androidTest/java/com/adobe/marketing/mobile/ConsentStatusChangeFunctionalTests.java

```diff
@@ -74,22 +79,33 @@ public void setup() throws Exception {
 		latch.await();
 
 		assertExpectedEvents(false);
+		mockNetworkService.reset();
 		resetTestExpectations();
 	}
 
+	@After
+	public void tearDown() {
+		mockNetworkService.reset();
+	}
+
 	// Test sendNetworkRequest(final EdgeHit edgeHit, final Map<String, String> requestHeaders, final int attemptCount)
 	@Test
 	public void testCollectConsent_whenNo_thenHits_hitsCleared() throws Exception {
 		// setup
 		updateCollectConsent(ConsentStatus.NO);
 		getConsentsSync();
+		mockNetworkService.reset();
 		resetTestExpectations();
```

Comment:
Let's add a reset util, if this is needed in multiple tests.

```
reset() {
    mockNetworkService.reset();
    resetTestExpectations();
}
```

Could also add mockNetworkService.reset() in the resetTestExpectations().